src/components/graph/menus/create-node-menu.js

```diff
@@ -284,11 +290,17 @@ const CreateNodeMenu = ({
                 },
             },
         },
+        UNBUILD_NODE: {
+            onRoot: false,
+            action: () => unbuildNode(),
+            id: 'unbuildNode',
+            disabled: !isNodeUnbuildingOrRemovingAllowed(),
```

Comment:
Unbuilding the node should also be disabled if the node is already disabled imo

Reply:
good point, fixed